extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -241,6 +244,28 @@ export class CreateSqlMigrationServiceDialog {
 			}
 		}).component();
 
+		this._createResourceGroupLink = this._view.modelBuilder.hyperlink().withProps({
+			label: constants.CREATE_NEW,
+			url: ''
+		}).component();
+
+		this._createResourceGroupLink.onDidClick(async e => {
```

Comment:
@alanrenmsft I've been thinking for a while that we should have Azure account based controls in the ModelView stuff so that we don't have to keep repeating this kind of logic across multiple extensions.

Anyways this is fine for now, but something to keep in mind that might be good to do next time someone wants to add "create new" functionality for a resource group or other resource.